src/main/java/com/nexmo/quickstart/Util.java

```diff
@@ -58,4 +59,29 @@ public static String envVar(String key) {
         }
         return value;
     }
+
+    /**
+     * Look up a required environment variable and throw an
+     * IllegalArgumentException if the value is not one of "0", "1", "true", "false", "on" or "off".
```

Comment:
From a cursory reading of javadoc best practices it looks like it's best to avoid using a `@throws` or `@exception` tag correct?